cmd/agent/app/agent.go

```diff
@@ -348,7 +349,7 @@ func startExecutionController(ctx controllerscontext.Context) (bool, error) {
 		EventRecorder:      ctx.Mgr.GetEventRecorderFor(execution.ControllerName),
 		RESTMapper:         ctx.Mgr.GetRESTMapper(),
 		ObjectWatcher:      ctx.ObjectWatcher,
-		PredicateFunc:      helper.NewExecutionPredicateOnAgent(),
+		PredicateFunc:      predicate.Funcs{},
```

Comment:
It looks weird to have an empty Predicate function.
I know the `PredicateFunc` is used to build the event filters by the work-status-controller, does it have to be set a dummy one?